src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests3.cs

```diff
@@ -1771,5 +1771,92 @@ static void M(int i)
                 Diagnostic(ErrorCode.ERR_AmbigUDConv, "new A()").WithArguments("A.implicit operator B(A)", "B.implicit operator B(A)", "A", "B").WithLocation(16, 43)
                 );
         }
+
+        [WorkItem(40714, "https://github.com/dotnet/roslyn/issues/40714")]
+        [Fact]
+        public void BadGotoCase_01()
```

Comment:
>BadGotoCase_01 [](start = 20, length = 14)

I think we should also test SemanticModel behavior: GetDeclaredSymbol, Get SymbolInfo, etc. The call stack in #40714 indicates that IDE requests information about declarations.

Reply:
Should probably test IOperation as well.

---
In reply to: [379658895](https://github.com/dotnet/roslyn/pull/41709#discussion_r379658895) [](ancestors = 379658895)